### DISCUSSION TITLE
Replaced cd command and nvm problems

### DISCUSSION THREAD

**Discussion by wiraki** [ajeetdsouza/zoxide#842] [Q&A]
With `eval "$(zoxide init --cmd cd zsh)"` in my .zshrc I am having trouble sourcing `nvm.sh` (https://github.com/nvm-sh/nvm/blob/master/nvm.sh) because: 

```
$  .nvm/nvm.sh debug
.nvm/nvm.sh: 33: cd: Illegal option -q
both the tree and the node path are required
```

Since no one else is complaining about this, I guess there is an issue in my configuration, but I don't know what. I was wondering if anyone can help.

zsh verison: `zsh 5.8.1`
zoxide version: `0.9.4`

**Comment by rogerneel** [comment]
Not sure if you figured this out, but regular `cd -q` is an invalid option:
```
> cd -q my_dir
bash: cd: -q: invalid option
cd: usage: cd [-L|[-P [-e]] [-@]] [dir]
```

**Reply by wiraki** [reply]
Hello, yes, I did figure it out in the end. Thank you! 

I see that the `nvm.sh` initialization script is trying to recognize if the shell is zsh, and then tries to use the `-q` flag.

I guess this problem does not arise if you are `eval`ing nvm before `eval`ing zoxide and changing the default cd command. But in my case I've moved to lazy-loading nvm only when it is needed, and by then cd is modified by zoxide.
